.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+# Mood Analysis
+Congratulations! You're submitting your assignment.
+
+## Comprehension Questions
+
+|  Question 	|  Answer 	|
+|---	|---	|
+|  What data structure did you use, an Array or a Hash?  Why?	|   	|
```

Comment:
I'm not sure if this makes sense since the instructions explicitly state to use an array